x-pack/solutions/security/plugins/security_solution/public/management/pages/policy/models/advanced_policy_schema.ts

```diff
@@ -2268,6 +2268,17 @@ export const AdvancedPolicySchema: AdvancedPolicySchemaType[] = [
       }
     ),
   },
+  {
+    key: 'windows.advanced.events.security.provider_etw',
+    first_supported_version: '8.19.0',
+    documentation: i18n.translate(
+      'xpack.securitySolution.endpoint.policy.advanced.windows.advanced.events.security.provider_etw',
+      {
+        defaultMessage:
+          'Controls whether Microsoft-Windows-Security-Auditing ETW provider is enabled for security events collection. Set to false to disable the provider. Default: true.',
```

Comment:
does the default behavior of Endpoint regarding the provider change between 8.18 and 8.19?

if yes, do we need a migration on existing policies to keep the existing behavior?

Reply:
awesome, thanks for the explanation!